imagej/imagej-ops2/src/main/java/net/imagej/ops2/map/neighborhood/MapNeighborhoodWithCenter.java

```diff
@@ -118,10 +120,20 @@ public void compute(final RandomAccessibleInterval<I> in1, final Shape in2,
 		RandomAccessibleInterval<Neighborhood<I>> neighborhoodInput = Views
 			.interval(in2.neighborhoodsRandomAccessibleSafe(in1), in1);
 
+		if (centerAwareOp instanceof RichOp) {
+			((RichOp)centerAwareOp).metadata().hints().plus(
+					BaseOpHints.History.SKIP_RECORDING);
+		}
+
 		LoopBuilder.setImages(neighborhoodInput, in1, out).multiThreaded()
 			.forEachPixel((neighborhood, inPixel, outPixel) -> {
 				centerAwareOp.compute(neighborhood, inPixel, outPixel);
 			});
+
+		if (centerAwareOp instanceof RichOp) {
+			((RichOp)centerAwareOp).metadata().hints().minus(
+					BaseOpHints.History.SKIP_RECORDING);
```

Comment:
If `SKIP_RECORDING` hint was already set before getting plussed in above... then should we still be minusing it out again here? Don't we just want to restore the hints back to their original state?